aggrivate/aggrivate.js

```diff
@@ -641,13 +641,13 @@ function loggit(lognum, message) {
             if (config.logging.log_errors_only == 1) {
                 return true;
             }
-            fd = fs.createWriteStream('/path/to/logs/' + config.folders.cg_log + '/' + config.logging.cg_acclog, {'flags': 'a'});
+            fd = fs.createWriteStream(config.folders.cg_log + '/' + config.logging.cg_acclog, {'flags': 'a'});
```

Comment:
I removed the hardcoded paths here as I thought the user can provide the full path inside of `cg_log`